packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -94,6 +95,11 @@ class PaginatedDataTable extends StatefulWidget {
     this.checkboxHorizontalMargin,
     this.controller,
     this.primary,
+    this.dataRowColor,
+    this.dividerThickness,
+    this.dataTextStyle,
+    this.headingRowColor,
+    this.headingTextStyle,
```

Comment:
Can you add tests that verify these new properties are all properly set when used?

Reply:
I will try this.